src/main/resources/hudson/views/ParameterFilter/config.jelly

```diff
@@ -9,15 +9,16 @@
       <f:textbox name="valueRegex" field="valueRegex" />
   </f:entry>
   <f:entry help="/plugin/view-job-filters/parameter-default-help.html">
-      <f:checkbox name="useDefaultValue" field="useDefaultValue"/> ${%Match the configured default values instead of the actual value from last successful build}

```

Comment:
name is unnecessary when field is used (kept in a couple of places elsewhere that it's being used in JavaScript)